### DISCUSSION TITLE
在繳交HW0時遇到的小問題

### DISCUSSION THREAD

**Discussion by dayu1004** [1101-GS4719-C/Python-Community-GS4719-C#472] [Q&A]
老師派下來的Github作業裡面有一個是嘗試修改內文之後要pull requests，讓助教知道我已經會使用這個功能了。
但我正要嘗試從fork開始操作的時候，卻發現fork那個地方說disable QQ
有看了老師在EE-class中的pull requests的教學影片，但老師也是很順利(?)的fork了。上網查了資料好像也沒有說到相關狀況怎麼解決，所以我一直卡在這邊。
真的有點苦惱，希望大家可以幫我一下(雖然這感覺是個小問題

**Comment by lx3211** [comment]
今天下課詢問過老師
老師說是題目敘述不好不用pull request
只要clone下來再push 上去
助教就知道完成作業了～

**Reply by dayu1004** [reply]
原來是這樣啊! 非常謝謝你!

**Comment by JunTingLin** [comment]
順帶一提😀
常常聽到發PR就是指pull request
可以參考這篇 [🌋誰說github儲存庫只能放code!目前兩個最火的儲存庫~ 我想要發PR成為貢獻者(誤) #492](https://github.com/NCU-GS4719-Python/Python-Community/discussions/492)
的第四點#如何成文開源專案的貢獻者

或是可以參考git大神高見龍的[與其它開發者的互動 - 使用 Pull Request（PR）](https://gitbook.tw/chapters/github/pull-request)
